### PR TITLE
Add Avatar iOS / Usage documentation

### PR DESCRIPTION
**Pull request by santigil**

### CODE REVIEW

File: www/src/pages/components/avatar/usage/index.mdx [hunk 1]
```diff
@@ -0,0 +1,141 @@
+---
+title: Avatar
```
Comment (santigil) [inline www/src/pages/components/avatar/usage/index.mdx:2]: @danoc This header doesn't render locally for some reason. 

🤔 Does it need to be defined after the imports or somehow differently?

Comment (danoc) [reply www/src/pages/components/avatar/usage/index.mdx:2]: It seems to be working in the deploy preview of this PR:

https://deploy-preview-703--thumbprint.netlify.app/

But I think I know what the issue is. I've created this to track it:
https://github.com/thumbtack/thumbprint/issues/706

**Comment by danoc** [comment]
I told @santigil this on Slack but I've also created #705 to track the syntax highlighting.

**Comment by danoc** [comment]
This LGTM once the screenshots are added. @kevinmbeaulieu, can you take a pass at the iOS contents in the doc?

File: www/src/cms/components-ios/avatar.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+<div>
+<b>Entity</b>: a company, business, or service. A square with our standard 4px border-radius is best here.
+</div>
+
+<div>
+<b>User</b>: a person or user. A circle is best suited for people - primarily faces.
+</div>
+
+insert iOS screenshots here
```
Comment (kevinmbeaulieu) [inline www/src/cms/components-ios/avatar.md:19]: Don't forget to add these

File: www/src/cms/components-ios/avatar.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+---
```
[the remark this answers is not in the document]
Comment (danoc) [reply www/src/cms/components-ios/avatar.md:1]: We can keep it here. I'll move it once I remove the CMS.

**Comment by danoc** [comment]
@santigil — Friendly reminder about this PR. The content is great but it's missing the screenshots.

**Review by kevinmbeaulieu — APPROVED** [review]

**Pull request merged**